<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="SUBHASHA-PC" name="com.jbk.owp.test.DashboardPageTest" tests="37" failures="6" timestamp="21 Jan 2020 12:02:24 GMT" time="11.502" errors="0">
  <testcase name="main_Navigation_Tab_Text_Check" time="0.269" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="Dashboard_Main_Heading_Text_Check" time="0.065" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="user_Navigation_spell_Check" time="0.120" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="oprators_Navigation_spell_Check" time="0.062" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="useful_Links_spell_Check" time="0.062" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="downloads_Navigation_Spelling_Check" time="0.222" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="logout_Navigation_Spelling_Check" time="0.061" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="Dashboard_Navigation_link_Check" time="0.077" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="user_Navigation_link_Check" time="0.069" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="oprators_Navigation_link_Check" time="0.120" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="usefulLinks_navigation_link_Check" time="0.056" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="downloads_Navigation_link_Check" time="0.082" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="sidebar_Menu_Navigations_Count" time="0.063" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="User_Navigation_isdisplayed_Check" time="0.164" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="operators_Navigation_isdisplayed" time="0.197" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="usefullinks_Navaigation_isdisplayed" time="0.123" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="downloads_navigation_isdisplayed" time="0.093" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="logouttabisdisplayed" time="0.084" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="moveover_Dashboard_Bgcolor" time="0.271" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="moveover_User_Bgcolor" time="0.220" classname="com.jbk.owp.test.DashboardPageTest">
    <failure type="java.lang.AssertionError" message="expected [rgba(30, 40, 44, 1)] but found [rgb(30, 40, 44)]">
      <![CDATA[java.lang.AssertionError: expected [rgba(30, 40, 44, 1)] but found [rgb(30, 40, 44)]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.jbk.owp.test.DashboardPageTest.moveover_User_Bgcolor(DashboardPageTest.java:203)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- moveover_User_Bgcolor -->
  <testcase name="moveover_Oprators_Bgcolor" time="0.364" classname="com.jbk.owp.test.DashboardPageTest">
    <failure type="java.lang.AssertionError" message="expected [rgba(30, 40, 44, 1)] but found [rgb(30, 40, 44)]">
      <![CDATA[java.lang.AssertionError: expected [rgba(30, 40, 44, 1)] but found [rgb(30, 40, 44)]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.jbk.owp.test.DashboardPageTest.moveover_Oprators_Bgcolor(DashboardPageTest.java:216)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- moveover_Oprators_Bgcolor -->
  <testcase name="moveover_UsefulLinks_Bgcolor" time="0.212" classname="com.jbk.owp.test.DashboardPageTest">
    <failure type="java.lang.AssertionError" message="expected [rgba(30, 40, 44, 1)] but found [rgb(30, 40, 44)]">
      <![CDATA[java.lang.AssertionError: expected [rgba(30, 40, 44, 1)] but found [rgb(30, 40, 44)]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.jbk.owp.test.DashboardPageTest.moveover_UsefulLinks_Bgcolor(DashboardPageTest.java:229)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- moveover_UsefulLinks_Bgcolor -->
  <testcase name="moveover_downloads_Bgcolor" time="0.490" classname="com.jbk.owp.test.DashboardPageTest">
    <failure type="java.lang.AssertionError" message="expected [rgba(30, 40, 44, 1)] but found [rgb(30, 40, 44)]">
      <![CDATA[java.lang.AssertionError: expected [rgba(30, 40, 44, 1)] but found [rgb(30, 40, 44)]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.jbk.owp.test.DashboardPageTest.moveover_downloads_Bgcolor(DashboardPageTest.java:242)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- moveover_downloads_Bgcolor -->
  <testcase name="moveover_Logout_Bgcolor" time="0.238" classname="com.jbk.owp.test.DashboardPageTest">
    <failure type="java.lang.AssertionError" message="expected [rgba(0, 0, 0, 0.1)] but found [rgba(0, 0, 0, 0)]">
      <![CDATA[java.lang.AssertionError: expected [rgba(0, 0, 0, 0.1)] but found [rgba(0, 0, 0, 0)]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.jbk.owp.test.DashboardPageTest.moveover_Logout_Bgcolor(DashboardPageTest.java:254)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- moveover_Logout_Bgcolor -->
  <testcase name="jbk_Main_Heading_Spell_check" time="0.091" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="jbk_subHeading_link" time="0.062" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="jbk_text_logo_isdisplayed" time="0.097" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="user_img_isdisplayed" time="0.190" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="dashboard_Main_Heading_Spell" time="0.066" classname="com.jbk.owp.test.DashboardPageTest">
    <failure type="java.lang.AssertionError" message="expected [Dashboard Courses Offered] but found [[[FirefoxDriver: firefox on WINDOWS (cbe88e0d-81c6-47f9-9186-e77c247d9054)] -&gt; xpath: //h1]]">
      <![CDATA[java.lang.AssertionError: expected [Dashboard Courses Offered] but found [[[FirefoxDriver: firefox on WINDOWS (cbe88e0d-81c6-47f9-9186-e77c247d9054)] -> xpath: //h1]]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:442)
at com.jbk.owp.test.DashboardPageTest.dashboard_Main_Heading_Spell(DashboardPageTest.java:293)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- dashboard_Main_Heading_Spell -->
  <testcase name="dashboard_Fotter_Spell" time="0.131" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="pullright_Heading_spell" time="0.146" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="selenium_Course_Spell" time="0.080" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="automationtest_spell" time="0.066" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="verifyUserLink" time="3.599" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="verifyOperatorslink" time="0.253" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="verifyUsefulLink" time="0.308" classname="com.jbk.owp.test.DashboardPageTest"/>
  <testcase name="verifyDownloadLink" time="2.629" classname="com.jbk.owp.test.DashboardPageTest"/>
</testsuite> <!-- com.jbk.owp.test.DashboardPageTest -->
